### PR TITLE
Support for the Expand op with constant shape inputs

### PR DESCRIPTION
**Pull request by tomdol**
This PR makes the unit tests and model tests pass when compiled with nGraph@master

### CODE REVIEW

File: onnxruntime/core/providers/ngraph/ngraph_execution_provider.cc [hunk 1]
```diff
@@ -210,6 +210,10 @@ static bool IsUnsupportedOpMode(const Node* node, const onnxruntime::GraphViewer
       return initializers.find(x_zero_point->Name()) == initializers.end() ||
              initializers.find(w_zero_point->Name()) == initializers.end();
     } // else -> xzp & wzp are 0 by default according to ONNX spec
+  } else if (optype == "Expand") {
+    // nGraph only supports constant shape input values
+    const auto& shape_input = node->InputDefs()[1];
+    return initializers.find(shape_input->Name()) == initializers.end();
```
Comment (skottmckay) [inline onnxruntime/core/providers/ngraph/ngraph_execution_provider.cc]: graph_utils::IsConstantInitializer would be preferable if possible. it checks parents graphs as well as for inputs that could override the initializer (making it non-constant).

Comment (tomdol) [reply onnxruntime/core/providers/ngraph/ngraph_execution_provider.cc]: The `graph_utils::IsConstantInitialize` helper requires a reference to `Graph` while in the provider I only have acces to the `GraphViewer`. I couldn't find a method in the viewer class that returns the original graph - is there a way to do it?

Comment (skottmckay) [reply onnxruntime/core/providers/ngraph/ngraph_execution_provider.cc]: I believe GraphViewer::IsConstantInitializer will do what you need and call the graph_utils method with the original graph inside the GraphViewer instance.

Comment (tomdol) [reply onnxruntime/core/providers/ngraph/ngraph_execution_provider.cc]: Thanks, that helped :) I've pushed the change already.

Should the same approach be used for other ops that we only support if some of their inputs are constants? I'll create a separate PR in this case.

**Pull request merged**